### PR TITLE
ZCS-11872 Fix condition to consider Zimbra 10 also for rolling upgrade

### PR DESCRIPTION
**Pull request by silentsakky**

### CODE REVIEW

**Review by k-kato — APPROVED** [review]

File: WebRoot/public/login.jsp [hunk 1]
```diff
@@ -224,16 +224,16 @@
                                 String versionPart = tokenParts[2];
                                 Map<?, ?> decodedTokenMap = TokenUtil.getAttrs(versionPart);
                                 String version = (String) decodedTokenMap.get("version");
-                                pageContext.setAttribute("isZ9Mailbox", version.startsWith("9"));
+                                pageContext.setAttribute("isUpgradedMailbox", version.startsWith("9") || version.startsWith("10"));
```
Comment (jiteshsojitra) [inline WebRoot/public/login.jsp:227]: version greater than >=9 by getting major so no need to update in future for version 11?

**Review by jiteshsojitra — APPROVED** [review]

File: WebRoot/public/login.jsp (hunk 1 above)
Comment (silentsakky) [reply WebRoot/public/login.jsp:227]: it's java code so I was not sure how to do that, let me check with someone

Comment (deepakzimb) [reply WebRoot/public/login.jsp:227]: boolean condition = Integer.parseInt(version) >= 9 ? true : false;
pageContext.setAttribute("isUpgradedMailbox", condition);

Comment (silentsakky) [reply WebRoot/public/login.jsp]: Fixed

**Review by deepakzimb — APPROVED** [review]

**Pull request merged**